### PR TITLE
ADD_All_Forms_end_SuccessfulRequest_Modal_Windows

### PR DESCRIPTION
**Pull request by ShlenchakBogdan**
ADD_All_Forms_end_SuccessfulRequest_Modal_Windows

### CODE REVIEW

File: src/Img/modal-selection-cancel.png
Comment (VladislavLiubasiuk) [inline src/Img/modal-selection-cancel.png:1]: we have this icon made in icomoon

**Review by Roman1518 — APPROVED** [review]

File: src/css/style.css [hunk 1]
```diff
@@ -1697,6 +1947,145 @@ body{
...
+        margin-top: 37px;
+    }
+    #AllPopUpForms .partnersPopUp-fields{
+        width: 217px;
+        margin-bottom: 20px;
+    }
+    #AllPopUpForms .input-block{
+        margin: 0;
+        width: 217px;
+    }
```
Comment (VladislavLiubasiuk) [inline src/css/style.css:2025]: I believe this can be shortened a bit

**Review by VladislavLiubasiuk — CHANGES_REQUESTED** [review]
redundant cancel, should be used through icomoon
minor issues in css, dublicating code

**Comment by VitaliyBogdanuk** [comment]
@ShlenchakBogdan conflict

**Pull request closed without merging**